### ISSUE TITLE
Bad error messaging on JSON configure files

### ISSUE THREAD

**Issue by radAragon** [myrmex-org/myrmex#80]
Hi, here's the problem:

I set my lambda config file as below:
```json
{
  "params": {
    "Timeout": 60,
    "MemorySize": "512",
    "Runtime": "nodejs6.10",
    "Handler": "lambda.handler",
    "Role": "arn:aws:iam::163140259062:role/cs-site-lambda-function",
  }
}
```

So, when I try to run `myrmex anything` the error is just this batch of words without meaning =P
```json
{
  "name": "myrmex",
  "hostname": "csrjl130l",
  "pid": 9679,
  "level": 60,
  "promise": {
    "isFulfilled": false,
    "isRejected": true,
    "rejectionReason": {}
  },
  "reason": {},
  "msg": "Unhandled Rejection",
  "time": "2017-07-13T17:13:53.111Z",
  "v": 0
}
```

After some time digesting the possible errors I found the extra comma after the "Role" attribute in config.json.

I think such a failure could be at least discernible as a JSON configuration file loading error.

Thanks, mate

**Comment by AlexisNo** [comment]
Indeed, some errors where swallowed. :confused: 

The PR https://github.com/myrmex-org/myrmex/pull/81 improves error logs in the console and in `myrmex.log`. :smile:

**Comment by AlexisNo** [comment]
Published in [version 0.2.2](https://github.com/myrmex-org/myrmex/blob/master/packages/core/CHANGELOG.md#022-2017-07-14) of `@myrmex/core` :tada:
